module/VuFindSearch/src/VuFindSearch/Response/SimpleRecord.php

```diff
@@ -58,6 +62,8 @@ class SimpleRecord implements RecordInterface
     public function __construct($fields)
     {
         $this->fields = $fields;
+        $this->sourceIdentifier = $this->searchBackendIdentifier
```

Comment:
Would a `setIdentifiers` call be more concise?

Reply:
Yes, fixed in all classes.